vuepress/src/.vuepress/configs/sidebar/en.ts

```diff
@@ -37,16 +37,17 @@ export const en: SidebarConfig = {
     {
       text: "Pre Reading Material",
       children: [
-        "000100_About_bcgov_cloud.md",
-        "000200_sea_at_bc_Gov.md",
-        "000300_procurement_value_proposition.md",
-        "000400_shared_responsibility_model.md",
-        "000500_Providing_an_sea_service.md",
-        "000600_billing.md",
-        "000700_communities_of_practise.md",
-        "000800_service_levels.md",
-        "000900_demo.md",
-        "001000_questions.md",
+        "000100_who_are_you.md",
```

Comment:
Capitalize the first word?  Ahh but this is a file name that no one will ever see.  So not worth making a fuss about a minor error